go.mod

```diff
@@ -119,17 +119,17 @@ require (
 	gopkg.in/ini.v1 v1.67.0
 	gopkg.in/yaml.v3 v3.0.1
 	helm.sh/helm/v3 v3.16.2
-	k8s.io/api v0.31.2
-	k8s.io/apiextensions-apiserver v0.31.2
-	k8s.io/apimachinery v0.31.2
-	k8s.io/cli-runtime v0.31.2
-	k8s.io/client-go v0.31.2
-	k8s.io/code-generator v0.31.2
-	k8s.io/component-base v0.31.2
-	k8s.io/endpointslice v0.31.1
+	k8s.io/api v0.32.0-beta.0
+	k8s.io/apiextensions-apiserver v0.32.0-beta.0
+	k8s.io/apimachinery v0.32.0-beta.0
+	k8s.io/cli-runtime v0.32.0-beta.0
+	k8s.io/client-go v0.32.0-beta.0
+	k8s.io/code-generator v0.32.0-beta.0
+	k8s.io/component-base v0.32.0-beta.0
+	k8s.io/endpointslice v0.32.0-beta.0
```

Comment:
Let's wait until k8s 0.32.0-rc.0 gets released.

Reply:
ah sure, it will be scheduled in 2 weeks IIRC. Marking this as draft.